api/candlepin-api-spec.yaml

```diff
@@ -2849,6 +2849,305 @@ paths:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Subscription pool with given ID could not be found
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+  /pools/{pool_id}/entitlements:
+    get:
+      description: Retrieve a list of entitlements for a pool
+      tags:
+        - Pools
+      operationId: getPoolEntitlements
+      parameters:
+        - in: path
+          name: pool_id
+          description: Pool ID
+          required: true
+          schema:
+            type: string
+      x-java-response:
+        type: Iterable
+        isContainer: true
+        security: []
+      responses:
+        200:
+          description: A list of entitlements
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/EntitlementDTO'
+        404:
+          description: Subscription pool with given ID could not be found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Subscription pool with given ID could not be found
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+  /pools/{pool_id}/entitlements/consumer_uuids:
+    get:
+      description: Retrieve a list of Consumer UUIDs attached to a pool. Available only to superadmins
+      tags:
+        - Pools
+      operationId: listEntitledConsumerUuids
+      parameters:
+        - in: path
+          name: pool_id
+          description: Pool ID
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: A list of consumer UUIDs
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+        404:
+          description: Subscription pool with given ID could not be found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Subscription pool with given ID could not be found
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+  /pools/{pool_id}/cert:
+    get:
+      description: Retrieves a Subscription Certificate
+      tags:
+        - Pools
+        - Certificates
+      operationId: getSubCert
+      parameters:
+        - in: path
+          name: pool_id
+          description: Pool ID
+          required: true
+          schema:
+            type: string
+      responses:
```

Comment:
Missing API response (HTTP 404) when "Pool ID or cert not found".

Reply:
Added.